posts/irl-aws-iam.md

```diff
@@ -0,0 +1,124 @@
+---
+authors:
+- Matt Reed
+tags:
+- AWS
+- IAM
+date: 2020-12-04T00:00:00.000Z
+title: "In Real Life - Amazon Web Services: Identity Access Management"
+image: https://raw.githubusercontent.com/ippontech/blog-usa/master/images/2020/12/iam_kitten.png
+---
+
+## A Quick Note about In Real Life
+
+For some reason, non-physical concepts have always confused a lot of people unfamiliar with the digital realm. Even those who do work in the IT-services industry can get tripped up by subtle trivialities. This is an attempt to distill the complexities down into more digestible ideas for everyone.
+
+## Identity Access Management a.k.a. Sharing is Caring
+
+To begin, someone who has never heard of "Identity Access Management" (or IAM for short) might be unsure about what it should do in the first place. The questions to ask here are:
+
+1. What is being worked on?
+1. Who is working on it?
```

Comment:
I'm not sure what you mean by these questions and how they're relevant to the reader at this point. Can you elaborate on what you're trying to get at?